WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -5,14 +5,14 @@
       <thead>
         <tr>
           <th class="event"><%= t 'competitions.results_table.event' %></th>
-          <th class="country-rank">NR</th>
-          <th class="continent-rank">CR</th>
-          <th class="world-rank">WR</th>
+          <th class="country-rank"><abbr title="Country rank">NR</abbr></th>
```

Comment:
nit - NR = National rank :)

Reply:
nit - Maybe even have "Rank" written with a capital xD

Reply:
Please use localised descriptions via the `t(random.translation.key)` feature like so:

`<abbr title="<%= t 'foo.bar' %>">NR</abbr>`

where `<%= %>` is a template that executes and inserts Ruby code into the HTML, and `foo.bar` has to be replaced with an appropriate translation key based on https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/locales/en.yml

Reply:
Since thse i18n keys don’t exist yet (even for English), what should I do about this? Is English the fallback, should I fill as many as I can in the locale yamls (which would be German, English and possibly French)?

Reply:
English is the fallback, so you only need to create new translation strings in the English YAML file. We have a network of translators (mostly Delegates and some local cubers) who are registered as "validated translators". They will be automatically notified when a new entry is added to the English YAML, asking them to submit translations.

Reply:
Done&pushed